pulsar-broker-common/pom.xml

```diff
@@ -65,6 +65,25 @@
       <artifactId>jjwt-impl</artifactId>
     </dependency>
 
+    <dependency>
```

Comment:
is it really necessary to add all of these third party deps ?
I wonder if it won't be better to create a separate module and let users drop the jar in pulsar/lib

if we want to go this way we have to verify all the licenses/NOTICE files and list the artifacts in https://github.com/apache/pulsar/tree/master/distribution/server/src/assemble and probably in https://github.com/apache/pulsar/tree/master/distribution/server/licenses

Reply:
The two `auth0` libraries were likely chosen because they have support for retrieving and deserializing `JWKS` (Json Web Key Sets) objects. The `io.jsonwebtoken` libraries currently used by the `AuthenticationProviderToken` do not have this support.

Reply:
> The two `auth0` libraries were likely chosen because they have support for retrieving and deserializing `JWKS` (Json Web Key Sets) objects. The `io.jsonwebtoken` libraries currently used by the `AuthenticationProviderToken` do not have this support.

Exactly, io.jsonwebtoken doesn't support the JWKS feature I need 😄

Reply:
di you think that we could remove "io.jsonwebtoken" and use everywhere this new library ?

Reply:
Maybe. I need to check if com.oauth0 supports the functionality needed for AuthenticationProviderToken

Reply:
thanks